Makefile

```diff
@@ -132,6 +132,10 @@ k8s-migrate:
 	$(shell kubectl --namespace ${DEIS_APP} get pods | grep ${DEIS_APP}-cmd | awk '{print $$1}') \
 	python manage.py migrate
 
+render-k8s-templates:
+	k8s/api-svc.yaml.template.sh > k8s/api-svc.yaml
+	rm k8s/api-svc.yaml.template.sh
```

Comment:
cool, we can also render text and pass it to stdin of `cat foo.json | kubectl create -f -` if needed.